client/app/assets/less/redash/ant.less

```diff
@@ -36,3 +42,35 @@
 .ant-dropdown-in-bootstrap-modal {
   z-index: 1050;
 }
+
+// Make Ant labels more Redash like
+.ant-form-item-label label {
+  color: inherit;
+  white-space: initial;
+  line-height: initial;
+  text-align: left;
+}
+
+// Fix ant input number showing duplicate arrows
+.ant-input-number-input::-webkit-outer-spin-button,
+.ant-input-number-input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
```

Comment:
Looking at the [InputNumer](https://ant.design/components/input-number/) page on chrome, I don't see a duplicate button. Then why is this happening in Redash?

Reply:
Might be related to our `antd` version, I'll test it once I try to update our version